### ISSUE TITLE
downloading models 403 forbidden

### ISSUE THREAD

**Issue by duffercn** [meta-llama/llama3#85]
when running the download.sh:

Resolving download6.llamameta.net (download6.llamameta.net)... 18.238.192.92, 18.238.192.128, 18.238.192.117, ...
Connecting to download6.llamameta.net (download6.llamameta.net)|18.238.192.92|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2024-04-20 14:01:02 ERROR 403: Forbidden.

**Comment by zhouyinn** [comment]
Same here

**Comment by multics** [comment]
Readme.md says:

```
Visit the [Meta Llama website](https://llama.meta.com/llama-downloads/) and register to download the model/s.

Once registered, you will get an email with a URL to download the models. You will need this URL when you run the download.sh script.
```

**Comment by juanliii** [comment]
anything new?

**Comment by Di-Zayn** [comment]
same

**Comment by RoyalMoon** [comment]
Same here

**Comment by HYTHYThythyt** [comment]
same here.

**Comment by lambortao** [comment]
Same here

**Comment by duffercn** [comment]
> Readme.md says:
> 
> ```
> Visit the [Meta Llama website](https://llama.meta.com/llama-downloads/) and register to download the model/s.
> 
> Once registered, you will get an email with a URL to download the models. You will need this URL when you run the download.sh script.
> ```

yeah, the 403 error happens after the registration and getting the url

**Comment by CharliedoD** [comment]
same here

**Comment by Vincent2571** [comment]
same here

**Comment by Dreamlittlecat** [comment]
same here

**Comment by jianfeng-phper** [comment]
Download successfully with VPN

**Comment by jellyfish45** [comment]
Same here

**Comment by alizaree** [comment]
I submitted a new url request and it worked with the new url.

**Comment by YanJiaHuan** [comment]
I pulled on aws, it worked, and then scp the checkpoint to my local machine.

**Comment by Killerofthecard** [comment]
Same here. I just got the URL from meta, but this error is still met....
